corehq/apps/users/templates/users/commcare_user_account_confirmed.html

```diff
@@ -0,0 +1,29 @@
+{% extends login_template %}
+{% load i18n %}
+{% load field_tags %}
+{% load hq_shared_tags %}
+{% js_entry_b3 'users/js/accept_invite' %}
+
+{% block title %}{% blocktrans %}Account Confirmed{% endblocktrans %}{% endblock title %}
+{% block tabs %}
+{% endblock %}
+
+{% block login-content %}
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="reg-form-container sign-in-container">
+          <div class="form-bubble form-bubble-lg">
+            <h3>{% trans "Now you can log into the mobile app." %}</h3>
```

Comment:
super nit:
```suggestion
            <h3>{% trans "You can now log into the mobile app." %}</h3>
```
echoing the success banner phrasing [here](https://github.com/dimagi/commcare-hq/blob/cf9f847192d17ccb79ef985003e8a08f10a3eac0/corehq/apps/users/views/mobile/users.py#L1582)